src/index.js

```diff
@@ -12,13 +12,19 @@ import Game from './js/game';
 $(document).ready( () => {
   const game = new Game();
   $('.start-btn').click( () => {
-    game.startRound();
+    if ($('#p1Name').val() !== '' && $('#p2Name').val() !== '' && $('#p3Name').val() !== '') {
+      game.startRound();
+    } else {
+      // NAME REQUIRED
+    }
   });
   
   $('.final-solution-btn').click( () => {
-    game.validateAnswer();
-    $('.input-solve, .final-solution-btn').hide();
-    $('.feedback').show();
+    if ($('.input-solve').val() !== '') {
```

Comment:
Nice thinking here too! I like the logic of not allowing them to click solve without having anything in the input box.